README.md

```diff
@@ -1,16 +1,40 @@
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fopenebs%2Frawfile-localpv.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fopenebs%2Frawfile-localpv?ref=badge_shield)
 
-RawFilePV
-===
+## RawFile-LocalPV
+The RawFile-LocalPV OpenEBS Data-Engine is a simialr but more flexible, yet more complex derivation of the LocalPV-Hostpath Data-Engine. <BR>
 
-Kubernetes LocalPVs on Steroids
+This are a few reasons to consider using node-based (rather than network-based) storage architetcure:
+> - Performance: Almost no network-based storage solution can keep up with baremetal disk performance in terms of IOPS/latency/throughput combined. And you’d like to get the best out of the SSD you’ve got!
+> - On-premise Environment: You might not be able to afford the cost of upgrading all your networking infrastructure, to get the best out of your network-based storage solution.
+> - Complexity: Network-based solutions are distributed systems. And distributed systems are not easy! You might want to have a system that is easier to understand and to reason about. Also, with less complexity, you can fix unpredicted issues more easily.
 
-Prerequisite
+### Using node-based storage
+The OpenEBS LocalPV-HostPath Data-Engine makes it pretty easy to automatically provision hostPath PVs and use them in your workloads. But, there are known limitations though:
+
+> [!IMPORTANT]
+> - You can’t monitor volume usage: There are hacky workarounds to run “du” regularly, but that could prove to be a performance killer, since it could put a lot of burden on your CPU and cause your filesystem cache to fill up. Not really good for a production workload.
+> - You can’t enforce hard limits on your volume’s size: Again, you can hack your way around it, with the same caveats.
+> - You are stuck with whatever filesystem your kubelet node is offering
+> - You can’t customize your filesystem:
```

Comment:
After `:` is there something meant to be there?

Reply:
Dunno. Seems like a good idea to end the sentence there. Replaced the `:` with a `.`.